src/lib/y2partitioner/refinements/filesystem_type.rb

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env ruby
+#
+# encoding: utf-8
+
+# Copyright (c) [2017] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "y2storage"
+
+module Y2Partitioner
+  module Refinements
+    # Temporary refinement for Y2Storage::Filesystems::Type. It should be
+    # replaced by a proper class or any other mechanism at some point.
```

Comment:
in FACT I prefer to have it as constant in format and mount or method, that decide if format UI should be offered